lib/auxiliary_data_computation.rb

```diff
@@ -13,10 +13,16 @@ def self.compute_concise_results
       %w(average ConciseAverageResults),
     ].each do |field, table_name|
       ActiveRecord::Base.transaction do
-        ActiveRecord::Base.connection.execute "DELETE FROM #{table_name}"
-        ActiveRecord::Base.connection.execute "ALTER TABLE #{table_name} AUTO_INCREMENT = 1"
+        temp_table_name = "#{table_name}_temp"
+
+        # Drop the temporary table if it exists
+        ActiveRecord::Base.connection.execute("DROP TABLE IF EXISTS #{temp_table_name}")
```

Comment:
Ideally we should never keep this temporary table around. Instead of checking and deleting here, it would be better to make sure the table is always removed at the end of the computation (even if it fails). IIRC we can wrap it like this:

```ruby
begin
  ...
ensure
  ActiveRecord::Base.connection.execute("DROP TABLE IF EXISTS #{temp_table_name}")
end
```

Reply:
Fixed, much nicer solution!

Reply:
For the record, running code in an `ensure` block doesn't actually guarantee it will happen (the ruby process could crash, someone could trip over a power cord in the data center, etc).

Does this code assume the temp table doesn't exist at the start? I glanced through it and it seems like it might. If I'm right, then the only truly safe option would be to drop the temp table at the start if it exists.

Reply:
If that happens, the next job would just fail and we would know something unexpected went wrong, which I'd say is fine in this case (given how unlikely those events are). But if we want to be absolutely sure, we could run in both places, yes :D